### DISCUSSION TITLE
How to do without Sprite Pro Extruder?

### DISCUSSION THREAD

**Discussion by valentingerard** [LeeOtts/Ender3v2-Klipper-Configs#7] [Q&A] [answered]
Hi, I want to install your custom files, what do I need to modify when I don't have a sprite extruder. Only a CR Touch.
I'm waiting on my Raspberry to install klipper but I prefer to get all the infos before trying to figure out later

**Comment by markusbrand** [comment, marked as answer]
Hi,
the difference is mainly in the Extruder section. I would use the official https://github.com/Klipper3d/klipper/blob/master/config/printer-creality-ender3-v2-2020.cfg as a starting point. The defaults are good for your standard setup. If you want to stay with Marlin I can also recommend: https://github.com/mriscoc/Ender3V2S1 - but of the course the cool kids run with Klipper ;-)

Here a BLtouch example for you:
[bltouch]
sensor_pin: ^PB1
control_pin: PB0
x_offset: -42
y_offset: -10
z_offset: 0

**Reply by valentingerard** [reply]
Thanks for your reply,
I managed to make it work earlier, but thanks anyway always good to know!
